addons/kafka/1.x/kafka-1.yaml

```diff
@@ -13,6 +13,8 @@ spec:
   kubernetes:
     minSupportedVersion: v1.16.0
   requires:
+    - matchLabels:
+        kubeaddons.mesosphere.io/name: kudo
```

Comment:
This is not needed, and in fact will cause deployments to fail. Kubeaddons now automates the Kudo controller.